### PR TITLE
Fix: README - Update

### PR DESCRIPTION
**Pull request by oxedions**
Update main README to better explain what the stack is made for, and what is supported.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,46 +9,65 @@
...
+## Resources
 
-The stack documentation is available on the BlueBanquise main website, in [documentation subfolder](https://bluebanquise.com/documentation/).
+### Files
 
-The stack repositories are available in the [repositories subfolder](https://bluebanquise.com/repository/).
+The stack is split over multiple repositories:
 
-## The name
+* :globe_with_meridians: **Core:** stack CORE roles and tools, dedicated to deploying and configuring servers, is hosted on github, in this [same repository](https://github.com/bluebanquise/bluebanquise).
```
Comment (btravouillon) [inline README.md:26]:
```suggestion
* :globe_with_meridians: **[Core](https://github.com/bluebanquise/bluebanquise):** the CORE of the stack, provides roles and tools to deploy and configure hosts.
```

File: README.md [hunk 2]
```diff
@@ -9,46 +9,65 @@
...
 
-You may wonder where this name comes from:
+* :star: **Community:** stack COMMUNITY roles and tools, dedicated to bring more specialized features on top of CORE, is hosted on github, in the [community repository](https://github.com/bluebanquise/community).
```
Comment (btravouillon) [inline README.md:28]:
```suggestion
* :star: **[Community](https://github.com/bluebanquise/community):** COMMUNITY roles and tools, provides additional features on top of CORE. The release cycle of COMMUNITY is different than CORE.
```

File: README.md [hunk 3]
```diff
@@ -9,46 +9,65 @@
...
+## Supported software environment
 
-* To be included if demand:
-  * Ubuntu
-  * Debian
+| Operating System family | Operating System distribution | Tested versions    | Notes                               |
+| ----------------------- | ----------------------------- | ------------------ | ----------------------------------- |
+| Red Hat                 |                               |                    |                                     |
+|                         | RHEL                          | 7.x, 8.x           | Should work on all 7 and 8 versions |
+|                         | CentOS                        | 7.x, 8.x           | Should work on all 7 and 8 versions |
```
[the remark this answers is not in the document]
Comment (oxedions) [reply README.md:47]: Yup, I will update that.

File: README.md [hunk 4]
```diff
@@ -9,46 +9,65 @@
...
+|                         | CentOS Stream                 | NA                 | In dev, targeted for next release   |
+|                         | Oracle Linux                  | NA                 | In dev, targeted for next release   |
+|                         | CloudLinux                    | NA                 | In dev, targeted for next release   |
+|                         | RockyLinux                    | NA                 | Waiting for distribution release    |
+|                         | Fedora                        | NA                 | In dev, targeted for next release   |
```
[the remark this answers is not in the document]
Comment (oxedions) [reply README.md:52]: It is already done in https://github.com/bluebanquise/bluebanquise/pull/542
I just need to adapt to current master.

File: README.md (hunk 1 above)
Comment (oxedions) [reply README.md:26]: Ok for me.

File: README.md (hunk 2 above)
Comment (oxedions) [reply README.md:28]: Ok for me.

**Review by btravouillon — APPROVED** [review]
I still have concerns about Fedora. There is a new release every 6 months, I'm not sure the project can follow this cycle.

**Comment by oxedions** [comment]
I understand that. Fedora is in fact a need in mi job. We use it a lot to debug recent hardware : often RHEL kernel crash at boot, and booting a Fedora with a recent kernel, we can check if issue is related to hardware or not.
I will try to keep up, and if not possible, we will remove Fedora.

**Pull request merged**